istresearch/mailroom
# P4-2031 - 5.6.1 Upstream Merge

# For the Reviewer

When this is complete, you should approve the PR via github.

## Summary
Merged 5.6.1 from upstream

#### Release Note
Merged 5.6.1 from upstream

#### Breaking Changes
None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

1. Use `istresearch/mailroom:ci-4.0.5-dev-P4-2031` for mailroom testing. 
2. Start an engage 4.0.5-dev stack, using the latest RP code from: https://github.com/istresearch/rapidpro/pull/135 and the latest courier: https://github.com/istresearch/courier/pull/41
3. Ensure you can search contacts from within the UI
4. Ensure you can load the flow editor and modify nodes
5. Ensure you can start a flow and send messages to contacts, as well as receive them.
